src/cli.ts

```diff
@@ -477,32 +479,33 @@ function pluralize(label: string, num: number) {
     [label] = label.split(' ');
     return num === 1 ? `${label} is` : `${label}s are`;
   }
-
   return num === 1 ? `${label}` : `${label}s`;
 }
 
-export function getFallbackEntryPointsOrExit(
-  argsEntrypoints: string[] | undefined,
-  config: Config,
-) {
-  let res = argsEntrypoints;
-  if (
-    (!argsEntrypoints || !argsEntrypoints.length) &&
-    config.apiDefinitions &&
-    Object.keys(config.apiDefinitions).length > 0
-  ) {
-    const dir = config.configFile ? dirname(config.configFile) : process.cwd();
-    res = Object.values(config.apiDefinitions).map((fileName) => resolve(dir, fileName));
-  } else if (argsEntrypoints && argsEntrypoints.length && config.apiDefinitions) {
-    res = res!.map((aliasOrPath) => config.apiDefinitions[aliasOrPath] || aliasOrPath);
-  }
+type Args = string[] | undefined
+const getDirectory = (config: Config) => config.configFile ? dirname(config.configFile) : process.cwd();
+const getAliasOrPath = (config: Config, aliasOrPath: string) => config.apiDefinitions[aliasOrPath] || aliasOrPath
+const isArray = (args: Args) => args && args.length
+const globOrString = (res: Args, config: Config) => {
```

Comment:
rename argument name, why is it `res`, also you can make its type just `string[]` so you won't have to use `!` later